### PR TITLE
test: test logging thread class.

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
Unit tests for LoggingThread and LoggingIO class.

### CODE REVIEW

File: tests/test_logger.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from mock import Mock
+from six.moves import StringIO
+
+from kobo.worker.logger import LoggingThread, LoggingIO
+from .utils import ArgumentIsInstanceOf
+
+
+class TestLoggingThread(unittest.TestCase):
+
+    @unittest.skipIf(six.PY3, 'Encoding errors on Python 3')
```
Comment (rohanpm) [inline tests/test_logger.py]: Would you be able to file a bug about that?

Comment (alexandrevicenzi) [reply tests/test_logger.py]: Yes, but on GitHub we already have a bug report about py3.

Comment (rohanpm) [reply tests/test_logger.py]: Then you can mention it in the text here.

Comment (alexandrevicenzi) [reply tests/test_logger.py]: Created issue #66 for this case.

**Pull request merged**